identity/app/views/consentJourney.scala.html

```diff
@@ -175,15 +175,35 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
 
 @confirmModal = {
     <section class="identity-forms-message">
-
+        @if(onlySubscribedToV1Newsletters.isEmpty) {
+            <h1 class="identity-header">Really?</h1>
+        }else {
+            <h1 class="identity-header">Are you sure?</h1>
+        }
         <div class="identity-forms-message__body">
-            You have not selected any newsletters or Guardian products and services.<br/>
-            This means you will stop receiving these emails from us. Are you sure you want to continue?
+            @if(onlySubscribedToV1Newsletters.isEmpty) {
+                Looks like you haven't selected any newsletters or Guardian products and services.
+                <br/>
+                <br/>
+                This means you'll stop recieving <b>all</b> emails from us other than service communication.
+                <br/>
+                <br/>
+                Keeping you informed is a critical part of our job.
+                <br/>
+                <br/>
+                <b>Are you sure you want to unsubscribe from all Guardian emails?</b>
+            }else {
+                You've chosen not to recieve the following emails from the Guardian:
+                <br/>
+                <div class="identity-consent-journey-modal-reminder"></div>
```

Comment:
nitpicky, but `identity-consent-journey-modal__reminder` would be better practice (as reminder is a children of `identity-consent-journey-modal`